### ISSUE TITLE
Tag page - All page content should be contained within landmarks 

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14838]
**Describe the bug**

For accessibility purposes, [all content should be contained within landmark regions](https://dequeuniversity.com/rules/axe/4.3/region?application=AxeChrome). On the tag page, e.g. https://dev.to/t/devops, the content in the left and right sidebars (desktop view) are not within any landmark:

![screenshot of the tag page for DEV's devops tag. The right sidebar and left sidebar content have a red box around them to highlight them](https://user-images.githubusercontent.com/20773163/135091999-9d1a3cc9-3351-4c15-8f0f-736a8819635a.png)

In this case, the content could reasonably exist within `<aside>` elements ( "complementary" landmark), since they support the main content, but are not essential to it, and make sense when separated from the main content.

NB: This refers to the large-screen 'desktop' view pictured above only. On smaller screens, the content is triggered as a pop-up menu. For the purposes of this issue we do not wish to change any of the markup for the pop-up menu.

**To Reproduce**

- Visit a tag's page
- In devtools, use the axe extension to check the page
- Notice the error "All page content should be contained by landmarks"

**Expected behavior**

- The error should not appear in an axe report
- When I navigate by landmarks using a screen reader, I should be able to select a landmark to navigate to this sidebar content

**Additional context**

There is a different class of accessibility issues associated with the smaller screen 'menu popup' view for this content. It will be captured in a separate issue, and needn't affect the fix here.

**Comment by metamoni** [comment]
Hey, @aitchiss, I'd like to work on this one if that's OK :)

**Comment by aitchiss** [comment]
Yeeha! That's awesome - I've assigned it to you now @metamoni 😄

**Comment by rhymes** [comment]
Closed by #14899
